app/assets/javascripts/application.js

```diff
@@ -2,6 +2,7 @@
 //= require turbolinks
 //= require i18n
 //= require jquery.ui.autocomplete
+//= require jquery.ui.spinner
```

Comment:
also, a small image / animated gif would be great to see the ui changes :) thanks!